index.html

```diff
@@ -0,0 +1,76 @@
+                    and other forms of discrimination.</li>
+
+                <li>King participated in and led marches for the right to vote, desegregation, labor rights, and other
+                    civil rights.[1] He oversaw the 1955 Montgomery bus boycott and later became the first president of
+                    the Southern Christian Leadership Conference (SCLC). As president of the SCLC, he led the
+                    unsuccessful Albany Movement in Albany, Georgia, and helped organize some of the nonviolent 1963
+                    protests in Birmingham, Alabama. King was one of the leaders of the 1963 March on Washington, where
+                    he delivered his "I Have a Dream" speech on the steps of the Lincoln Memorial. The civil rights
+                    movement achieved pivotal legislative gains in the Civil Rights Act of 1964, Voting Rights Act of
+                    1965, and the Fair Housing Act of 1968.</li>
+
+                <li>King participated in and led marches for the right to vote, desegregation, labor rights, and other
+                    civil rights.[1] He oversaw the 1955 Montgomery bus boycott and later became the first president of
+                    the Southern Christian Leadership Conference (SCLC). As president of the SCLC, he led the
+                    unsuccessful Albany Movement in Albany, Georgia, and helped organize some of the nonviolent 1963
+                    protests in Birmingham, Alabama. King was one of the leaders of the 1963 March on Washington, where
+                    he delivered his "I Have a Dream" speech on the steps of the Lincoln Memorial. The civil rights
+                    movement achieved pivotal legislative gains in the Civil Rights Act of 1964, Voting Rights Act of
+                    1965, and the Fair Housing Act of 1968.</li>
+
+                <li>King participated in and led marches for the right to vote, desegregation, labor rights, and other
+                    civil rights.[1] He oversaw the 1955 Montgomery bus boycott and later became the first president of
+                    the Southern Christian Leadership Conference (SCLC). As president of the SCLC, he led the
+                    unsuccessful Albany Movement in Albany, Georgia, and helped organize some of the nonviolent 1963
+                    protests in Birmingham, Alabama. King was one of the leaders of the 1963 March on Washington, where
+                    he delivered his "I Have a Dream" speech on the steps of the Lincoln Memorial. The civil rights
+                    movement achieved pivotal legislative gains in the Civil Rights Act of 1964, Voting Rights Act of
+                    1965, and the Fair Housing Act of 1968.</li>
+            </ul>
+
+            <a href="https://en.wikipedia.org/wiki/Martin_Luther_King_Jr." id="tribute-link" target="_blank">Tribute
+                Link</a>
+        </section>
+    </div>
+
```

Comment:
Please avoid using blank spaces in your code. because you have to add blank lines to separate large or logical code blocks. this one is a small block